lib/rules/string-quotes/index.js

```diff
@@ -191,7 +191,6 @@ const rule = (primary, secondaryOptions, context) => {
 
 			if (isAtRule(node) && node.name === 'charset') {
 				// allow @charset rules to have double quotes, in spite of the configuration
-				// TODO: @charset should always use double-quotes, see https://github.com/stylelint/stylelint/issues/2788
```

Comment:
Could you revert this change since `string-quotes` has been removed in the `v16` branch?

Reply:
Iv created an issue on elirasza/stylelint-stylistic.
Since this won't be ever be tackled by us, the TODO can be safely removed.

Reply:
If we changed this file, we would need to resolve conflicts when rebasing or merging into the `v16` branch. It's just a pain, and I'd like to avoid extra work (see #7002).

Reply:
I see; this sucks.
Hopefully the ESM migration that you—for the most part—undertook, will come to an end soon.
Ill fix that later on today.